### PR TITLE
Use QGIS master

### PR DESCRIPTION
**Pull request by domi4484**

### CODE REVIEW

File: vcpkg/ports/qgis/portfile.cmake [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+message("VERSION: ${VERSION}")
+
+if(${VERSION} STREQUAL "master")
+    set(QGIS_REF ${VERSION})
+else()
+    string(REPLACE "." "_" TAG ${VERSION})
+    set(QGIS_REF "final-${TAG}")
+endif()
+
 set(QGIS_SHA512 d8075b98efe8ebea1ee53273b9427e0a7329ba8f1a96258d962dee52b0c5c08be2bea10b2130fee4c0acbc3f4a94b94da0033e8b6c92857e0c679c051545d3d8)
```
Comment (m-kuhn) [inline vcpkg/ports/qgis/portfile.cmake:11]:
```suggestion
set(QGIS_SHA512 c3667497aa5de622402c23577b80ce1b2eb4a2ce443c0f1ea8db769b30b69edd90abf688d3c4ae77a2f0f4d22be3e18c21a66be180712f5ff3995ee663754135)
```

from the build log

**Comment by m-kuhn** [comment]
@domi4484 you can also always compare https://github.com/opengisch/QField/tree/master/vcpkg/ports/qgis

**Comment by m-kuhn** [comment]
QtSomemodule ships a vendored copy of poly2tri still... My 2 commits can be reverted and we still need a poly2tri patch for qgis.
This can only be cleaned with qt6 (or a bigger patch for qt which is not what we want...)

**Comment by domi4484** [comment]
@m-kuhn building against QGIS master is not solving the PDF export issue. And is introducing some layout troubles, the kadas tools menus are looking strange. So lets put this to draft for the moment

**Pull request closed without merging**